src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: flex;
+  padding-right: 243px;
+}
+
+@media (min-width: 1024px) {
+  .header {
+    padding: 0 50px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .header-logo {
+    padding-right: 419px;
```

Comment:
You have used a fixed padding value for `.header-logo` inside a media query. Depending upon the content and design requirements, fixed values can cause overflow or accidental scroll bars. Unless necessary, it's better to use relative units like percentages or use CSS Grid or Flexbox for layout arrangements.